packages/svelte/src/internal/client/dom/template.js

```diff
@@ -48,7 +48,7 @@ export function template(content, flags) {
 		}
 
 		var clone = /** @type {TemplateNode} */ (
-			use_import_node ? document.importNode(node, true) : node.cloneNode(true)
+			use_import_node || is_firefox ? document.importNode(node, true) : node.cloneNode(true)
```

Comment:
Shouldn't this be on line 28 instead?

Reply:
Uh yeah it might be better

Reply:
Done

Reply:
Actually this might be too soon. Is `is_firefox` true at this point?

Reply:
Uh you are right... it's initialized in `init_operations` so it might be too soon indeed. I need to check the other usage too